apps/builder/app/dashboard/loading-state/loading.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react";
+import { Flex, css } from "@webstudio-is/design-system";
+import { SvgLoading } from "~/dashboard/loading-state/svg-loading";
+import { WebmLoading } from "~/dashboard/loading-state/webm-loading";
+
+const containerStyle = {
+  position: "absolute",
+  left: "0",
+  right: "0",
+  top: "0",
+  bottom: "0",
+};
+
+const lightContainerStyle = css(containerStyle, {
+  backgroundColor: "rgb(255 255 255 / 50%);",
+});
+const darkContainerStyle = css(containerStyle, {
+  backgroundColor: "rgb(0 0 0 / 80%);",
```

Comment:
needs a design token from design system @markxtang I am assuming you want to provide a design token for the background behind spinner?

Reply:
Like @markxtang said, the background color for the overlay is not in the design tokens, so I just copy the value from the figma.
![Screen Shot 2023-05-08 at 15 37 12](https://user-images.githubusercontent.com/32300655/236764703-3321a3e5-ca99-463d-a119-4c82775b9480.png)